lib/src/layer/tile_layer/tile_layer.dart

```diff
@@ -345,10 +344,14 @@ class _TileLayerState extends State<TileLayer> with TickerProviderStateMixin {
   StreamSubscription<TileUpdateEvent>? _tileUpdateSubscription;
   Timer? _pruneLater;
 
-  late final _resetSub = widget.reset?.listen((_) {
-    _tileImageManager.removeAll(widget.evictErrorTileStrategy);
-    if (mounted) _loadAndPruneInVisibleBounds(MapCamera.of(context));
-  });
+  StreamSubscription<void>? _resetSub;
```

Comment:
Stream needs to get canceled in dispose().

Reply:
Ah, forgot to do this, but did it in update :D

Reply:
Stream is already cancelled in `dispose`:  https://github.com/fleaflet/flutter_map/blob/8136ef0fc0b75b759ea5e02eeaa2e1eac3dca245/lib/src/layer/tile_layer/tile_layer.dart#L469

Reply:
My bad sorry, haven't seen it while reviewing.